lib/srv/discovery/fetchers/aws-sync/eks.go

```diff
@@ -58,6 +60,7 @@ func (a *Fetcher) pollAWSEKSClusters(ctx context.Context, result *Resources, col
 		result.EKSClusters = output.clusters
```

Comment:
pollAWSEKSClusters is a function that returns a function that fetches
 any chance that could be done with one less indirection?